### PR TITLE
Shmuel doc

### PR DESCRIPTION
**Pull request by ShmuelSfez**
fix comments

### CODE REVIEW

File: docs/fabric/MAS_router/mas_router.md [hunk 1]
```diff
@@ -81,4 +81,46 @@ Data Flow Description:
...
+- It becomes more pronounced in networks with bursty traffic patterns or when there are disparities in the arrival rates of packets destined for different output ports.
+
+In our design, HOL blocking can occur if we naively insert only one fifo_arb for each router (to save space), with the output connected to other tiles. In this setup, if there is a transaction destined for the north, and behind it, there is a transaction bound for the south, the south transaction gets blocked by the north transaction. In other words, if the north tile is not ready, all other transactions will be stuck behind it.
+
+The router module that addresses the HOL blocking problem by efficiently managing the allocation of communication resources among multiple incoming requests. Here's how the router module achieves this:
+
+1. **Input Buffering**:
+   - Incoming communication requests from neighboring tiles are first buffered in FIFO buffers within the `fifo_arb` module. Each FIFO buffer corresponds to a particular communication channel or direction.
+
+2. **Arbitration**:
```
Comment (amichai-bd) [inline docs/fabric/MAS_router/mas_router.md:105]: I think it's important to note that we have a more advanced Round Robin.
We are masking the RR candidates with 2 conditions:
- There is a valid request waiting to pop (this is the common practice)
- The target of this specific request is "ready" to accept new requests.

This information should be detailed in length in the fifo_arb.
Here it should be mentioned with a pointer for more details in the fifo_arb

File: docs/fabric/MAS_router/mas_next_tile_fifo_arb.md [hunk 2]
```diff
@@ -42,10 +42,24 @@ Data Flow Description:
 3. It determines the priority of different cardinal directions based on the target address and the current tile's ID.
 4. The selected cardinal direction for each request is then provided as output.
```
Comment (amichai-bd) [inline docs/fabric/MAS_router/mas_next_tile_fifo_arb.md:43]: if i rememeber correctly, this is the block where the routing algorithm actually takes place.
Currently we are using a "x,y" aproch, setting the next tiles to reach the 'x' and then reach the 'y'
In future we may want to improve this by having a different routing scheme. (Example responses will readh 'y' and then 'x'?)

File: docs/fabric/MAS_fabric/mas_fabric.md [hunk 3]
```diff
@@ -37,10 +45,9 @@ The `fabric` module operates in a single mode, facilitating data routing and com
...
 4. Local tile IDs are generated based on tile positions for routing.
 
-## 5. Configuration and Control
+## 5. different configuration
 
-**Configuration Registers:**
-The `fabric` module does not include explicit configuration registers. Configuration and control may be implemented at a higher system level.
+In future we will be able to cange the size of the fabric by changing the ROW and COL paramters, we will also be able to connect different tiles and not just the mini_core_tile.
 
 ## 6. Testing and Verification
```
Comment (amichai-bd) [inline docs/fabric/MAS_fabric/mas_fabric.md:53]: Also here -> please at least mention the coverage points , what needs to be tested. not how its tested.
(back pressure, full bandwidth, stress tests)

**Review by amichai-bd — APPROVED** [review]

File: docs/fabric/MAS_router/mas_router.md (hunk 1 above)
Comment (ShmuelSfez) [reply docs/fabric/MAS_router/mas_router.md:105]: done

File: docs/fabric/MAS_router/mas_next_tile_fifo_arb.md (hunk 2 above)
Comment (ShmuelSfez) [reply docs/fabric/MAS_router/mas_next_tile_fifo_arb.md:43]: done

File: docs/fabric/MAS_fabric/mas_fabric.md (hunk 3 above)
Comment (ShmuelSfez) [reply docs/fabric/MAS_fabric/mas_fabric.md:53]: done

**Pull request merged**